src/TreeSupport.cpp

```diff
@@ -2023,10 +2023,8 @@ void TreeSupport::filterFloatingLines(std::vector<Shape>& support_layer_storage)
                 return;
             }
 
-            Shape outer_walls = TreeSupportUtils::toPolylines(support_layer_storage[layer_idx - 1].getOutsidePolygons())
-                                    .createTubeShape(
-                                        closing_dist,
-                                        0); //.unionPolygons(volumes_.getCollision(0, layer_idx - 1, true).offset(-(config.support_line_width+config.xy_min_distance)));
+            const Shape relevant_forbidden = volumes_.getCollision(0, layer_idx, true);
```

Comment:
`getCollision` returns a reference, so `relevant_forbidden` could also be a reference and save a copy.

Reply:
It's just a code sanitization thing, if we don't fix it now we'll never touch it 😛

It's incredibly low-risk -- does anyone mind if I push this change directly to 5.8? It should change nothing except speed up the tree stuff a tiny bit, since we avoid a copy.

Reply:
I would say make a quick test and push to 5.8 :+1:

Reply:
Done!